docs/authentication/saml/okta.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: Okta Workforce
+description: Learn how to integrate Okta Workforce with Clerk using SAML SSO.
+---
+
+# Okta Workforce
+
+<TutorialHero
+  beforeYouStart={[
+    {
+      title: "Set up a Clerk application",
+      link: "/docs/quickstarts/setup-clerk",
+    },
+    {
+      title: "Add the appropriate Clerk SDK to your application",
+      link: "/docs/quickstarts/overview",
+    },
+  ]}
+>
+
+- Use Okta Workforce to enable single sign-on (SSO) via SAML for your Clerk application.
+
+</TutorialHero>
+
+Integrating Okta Workforce with Clerk allows you to use it as an Identity Provider (IdP) for your Clerk application to authenticate your users and manage their access to your application via SAML SSO.
+
+## Tutorial
+
+<Steps>
+
+### Set up Enterprise Connection in Clerk
+
+To create a SAML connection in Clerk:
+
+1. Navigate to the [Clerk Dashboard](https://dashboard.clerk.com).
+2. In the navigation sidebar, select **User & Authentication > [Enterprise Connections](https://dashboard.clerk.com/last-active?path=user-authentication/enterprise-connections)**.
+3. Select the **Create connection** button.
+4. You will be presented with a modal to create a new connection. Fill in the required fields.
+- **Name** - A unique name for the connection. This is so you can tell your connections apart.
```

Comment:
To nest this bulleted list, it needs to be double-indented. It'll look like this:
![image](https://github.com/clerk/clerk-docs/assets/5156915/34ad47a0-6d28-47ec-b350-a7d24f4401d5)

Instead of this:
![image](https://github.com/clerk/clerk-docs/assets/5156915/e444d515-f76a-40d7-a246-4afdc12a03c2)


Also, IMO the only item that should be here is the instructions for choosing Okta Workforce under Identity Provider. The other two fields should be understandable within the UX, and if not we should make suggestions to the dashboard team so the name and domain field's purposes are clearer

Reply:
Okay I'll remove the bullet points and change up the instruction.